flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/Executor.java

```diff
@@ -79,13 +79,23 @@
 	/**
 	 * Lists all registered catalogs.
 	 */
-	List<String> listCatalogs(String sessionid) throws SqlExecutionException;
+	List<String> listCatalogs(String sessionId) throws SqlExecutionException;
+
+	/**
+	 * List current catalog.
+	 */
+	String listCurrentCatalog(String sessionId) throws SqlExecutionException;
```

Comment:
we should not introduce new methods, instead, we can call `executeSql` directly

Reply:
@fsk119 I think you can create a new PR for cleaning methods in `Executor` first...

Reply:
https://github.com/apache/flink/pull/12923

Reply:
I don't notice Executor has `executeSql`  method... 
Sounds good to clean up these dulicate method as my next work!

Reply:
> I don't notice Executor has `executeSql` method...
> Sounds good to clean up these dulicate method as my next work!

I had opened a pr last week